tests/functional/AppCest.php

```diff
@@ -318,4 +319,17 @@ public function dynamicRouterDeniesAccessToInactiveRoutable(FunctionalTester $I)
         $I->dontSee('Hello World');
         $I->cantSeeEventTriggered(RouteMatched::class);
     }
+
+    public function carbonDatesAreTranslated(FunctionalTester $I)
```

Comment:
We need to test `MultiLanguage` middleware not Carbon

Reply:
Done.